cmd/kube-apiserver/app/server.go

```diff
@@ -430,6 +430,7 @@ func BuildGenericConfig(s *options.ServerRunOptions) (*genericapiserver.Config,
 			versionedInformers.Core().V1().Services().Lister(),
 		)
 	}
+	genericConfig.SharedInformerFactory = versionedInformers
```

Comment:
It's always suspicious if a line is forgotten and nothing break. Either it's unnecessary or tests are bad.